src/hidra/receiver/plugins/asapo_producer.py

```diff
@@ -339,37 +283,14 @@ def _parse_file_name(self, path):
             self.log.debug("file_regex: %s", self.file_regex)
             raise utils.UsageError("Does not match file pattern")
```

Comment:
This probably shouldn't be an exception. Files that don't match should simply be ignored and an info (or at most a warning) be logged.

Reply:
Changed.